authors/apps/articles/serializers.py

```diff
@@ -74,8 +73,9 @@ class ArticleSerializer(serializers.ModelSerializer):
     tagList = TagRelatedField(many=True, required=False, source='tags')
     favorited = serializers.SerializerMethodField(method_name="is_favorited")
     favoriteCount = serializers.SerializerMethodField(
-                            method_name='get_favorite_count'
-                            )
+        method_name='get_favorite_count'
+    )
+    bookmarked = serializers.SerializerMethodField(method_name="is_bookmarked")
```

Comment:
Does this mean you can bookmark disliked comments?

Reply:
Yes, you can @ridgekimani, bookmarking acts as a 'temporary' storage for articles you may want to re-read in future.